streams/src/test/java/org/apache/kafka/streams/state/internals/ChangeLoggingKeyValueBytesStoreTest.java

```diff
@@ -84,12 +67,9 @@ public void after() {
     public void shouldWriteKeyValueBytesToInnerStoreOnPut() {
         store.put(hi, there);
         assertThat(inner.get(hi), equalTo(there));
-    }
-
-    @Test
-    public void shouldLogChangeOnPut() {
-        store.put(hi, there);
-        assertThat(sent.get(hi), equalTo(there));
+        assertThat(collector.collected().size(), equalTo(1));
```

Comment:
Note here we collapsed a pair of tests (on the store, and one the underlying changelog) into one. Also for assertion we use a list instead of a map to validate overwritten changelog entries. Ditto below.